sphinx/source/docs/dev_guide/models.rst

```diff
@@ -49,42 +50,42 @@ Models contain properties, which are class attributes of type
 
 The `IntProps` model represents objects that have three integer values,
 ``prop1``, ``prop2``, and ``prop3``, that can be automatically serialized
-from python, and unserialized by BokehJS.
+from Python, and unserialized by BokehJS.
 
 .. note::
     Technically, ``prop1`` isn't an instance of ``Int``, but ``HasFields`` uses a
     metaclass that automatically instantiates `Property` classes when necessary,
     so ``prop1`` and ``prop2`` are equivalent (though independent) properties.
     This is useful for readability; if you don't need to pass any arguments to
-    property's constructor then prefer the former over the later.
+    property's constructor, then prefer the former over the later.
 
-There is wide variety of property types, ranging from primitive types such as:
+There is a wide variety of property types, ranging from primitive types such as:
 
 * :class:`~bokeh.core.properties.Byte`
 * :class:`~bokeh.core.properties.Int`
 * :class:`~bokeh.core.properties.Float`
 * :class:`~bokeh.core.properties.Complex`
 * :class:`~bokeh.core.properties.String`
 
-As well as container-like properties, that take other Properties as parameters:
+as well as container-like properties, that take other Properties as parameters:
```

Comment:
nit: I am not sure why Properties should be capitalized here

Reply:
It should just not be capitalized, I think. Thanks for pointing that out!